From: Qingqing <wangyiqing@kylinos.cn>
Date: Tue, 21 Jul 2026 07:52:43 +0000
Subject: Translated using Weblate (Lao)

Currently translated at 100.0% (60 of 60 strings)

Translation: openkylin-huanghe/qt5-ukui-filedialog
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/qt5-ukui-filedialog/lo/
---
 .../translations/qt5-ukui-filedialog_lo.ts         | 120 ++++++++++-----------
 1 file changed, 60 insertions(+), 60 deletions(-)

diff --git a/qt5-ukui-filedialog/translations/qt5-ukui-filedialog_lo.ts b/qt5-ukui-filedialog/translations/qt5-ukui-filedialog_lo.ts
index 9700a4d..e99e88f 100644
--- a/qt5-ukui-filedialog/translations/qt5-ukui-filedialog_lo.ts
+++ b/qt5-ukui-filedialog/translations/qt5-ukui-filedialog_lo.ts
@@ -6,17 +6,17 @@
     <message>
         <location filename="../kyfiledialog.cpp" line="3081"/>
         <source>Open File</source>
-        <translation type="unfinished"></translation>
+        <translation>ເປີດ</translation>
     </message>
     <message>
         <location filename="../kyfiledialog.cpp" line="3082"/>
         <source>Save File</source>
-        <translation type="unfinished"></translation>
+        <translation>ບັນທຶກ</translation>
     </message>
     <message>
         <location filename="../kyfiledialog.cpp" line="3095"/>
         <source>All Files (*)</source>
-        <translation type="unfinished"></translation>
+        <translation>ທັງໝົດ(*)</translation>
     </message>
 </context>
 <context>
@@ -24,22 +24,22 @@
     <message>
         <location filename="../file-operation-dialog/kyfiledialogrename.cpp" line="46"/>
         <source>bytes</source>
-        <translation type="unfinished"></translation>
+        <translation>ໄບຕ໌</translation>
     </message>
     <message>
         <location filename="../file-operation-dialog/kyfiledialogrename.cpp" line="47"/>
         <source>Bytes</source>
-        <translation type="unfinished"></translation>
+        <translation>ໄບຕ໌</translation>
     </message>
     <message>
         <location filename="../file-operation-dialog/kyfiledialogrename.cpp" line="50"/>
         <source>character</source>
-        <translation type="unfinished"></translation>
+        <translation>ຕົວອັກສອນ</translation>
     </message>
     <message>
         <location filename="../file-operation-dialog/kyfiledialogrename.cpp" line="51"/>
         <source>Character</source>
-        <translation type="unfinished"></translation>
+        <translation>ຕົວອັກສອນ</translation>
     </message>
     <message>
         <source>Saving &quot;%1&quot;</source>
@@ -48,7 +48,7 @@
     <message>
         <location filename="../file-operation-dialog/kyfiledialogrename.cpp" line="70"/>
         <source>Saving failed, the reason is: %1</source>
-        <translation type="unfinished"></translation>
+        <translation>ບັນທຶກບໍ່ສຳເລັດ, ເຫດຜົນ: %1</translation>
     </message>
     <message>
         <source>Filename too long</source>
@@ -57,7 +57,7 @@
     <message>
         <location filename="../file-operation-dialog/kyfiledialogrename.cpp" line="298"/>
         <source>Description: Save long file names to &quot;</source>
-        <translation type="unfinished"></translation>
+        <translation>ຄຳອະທິບາຍ: ບັນທຶກໄຟລ໌ທີ່ມີຊື່ຍາວໄປຍັງ &quot;</translation>
     </message>
     <message>
         <source>Copying &quot;%1&quot;</source>
@@ -90,50 +90,50 @@
     <message>
         <location filename="../file-operation-dialog/kyfiledialogrename.cpp" line="91"/>
         <source>Truncation</source>
-        <translation type="unfinished"></translation>
+        <translation>ຕັດ</translation>
     </message>
     <message>
         <location filename="../file-operation-dialog/kyfiledialogrename.cpp" line="92"/>
         <source>Save to long file name directory</source>
-        <translation type="unfinished"></translation>
+        <translation>ບັນທຶກໄປຍັງໄຟລ໌ທີ່ມີຊື່ຍາວ</translation>
     </message>
     <message>
         <location filename="../file-operation-dialog/kyfiledialogrename.cpp" line="93"/>
         <source>Rename</source>
-        <translation type="unfinished"></translation>
+        <translation>ປ່ຽນຊື່</translation>
     </message>
     <message>
         <location filename="../file-operation-dialog/kyfiledialogrename.cpp" line="286"/>
         <source>modify the name</source>
-        <translation type="unfinished"></translation>
+        <translation>ປ່ຽນຊື່</translation>
     </message>
     <message>
         <location filename="../file-operation-dialog/kyfiledialogrename.cpp" line="287"/>
         <location filename="../file-operation-dialog/kyfiledialogrename.cpp" line="293"/>
         <source>.</source>
-        <translation type="unfinished"></translation>
+        <translation>.</translation>
     </message>
     <message>
         <location filename="../file-operation-dialog/kyfiledialogrename.cpp" line="288"/>
         <source>Explanation: When renaming a file name, ensure it is within %1 %2 and </source>
-        <translation type="unfinished"></translation>
+        <translation>ຄຳອະທິບາຍ: ຜູ້ໃຊ້ປ່ຽນຊື່ໄຟລ໌, ຮັບປະກັນວ່າຢູ່ໃນ %1 %2, ຫຼັງຈາກນັ້ນ </translation>
     </message>
     <message>
         <location filename="../file-operation-dialog/kyfiledialogrename.cpp" line="126"/>
         <source>All applications</source>
-        <translation type="unfinished"></translation>
+        <translation>ໃຊ້ກັບທັງໝົດ</translation>
     </message>
     <message>
         <location filename="../file-operation-dialog/kyfiledialogrename.cpp" line="70"/>
         <source>The file name is too long. </source>
-        <translation type="unfinished"></translation>
+        <translation>ຊື່ໄຟລ໌ຍາວໂພດ </translation>
     </message>
     <message>
         <location filename="../file-operation-dialog/kyfiledialogrename.cpp" line="130"/>
         <location filename="../file-operation-dialog/kyfiledialogrename.cpp" line="172"/>
         <location filename="../file-operation-dialog/kyfiledialogrename.cpp" line="239"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>ຍົກເລີກ</translation>
     </message>
     <message>
         <source>Apply</source>
@@ -146,24 +146,24 @@
     <message>
         <location filename="../file-operation-dialog/kyfiledialogrename.cpp" line="231"/>
         <source>Front truncation</source>
-        <translation type="unfinished"></translation>
+        <translation>ຕັດຕົ້ນ</translation>
     </message>
     <message>
         <location filename="../file-operation-dialog/kyfiledialogrename.cpp" line="232"/>
         <source>Post truncation</source>
-        <translation type="unfinished"></translation>
+        <translation>ຕັດທ້າຍ</translation>
     </message>
     <message>
         <location filename="../file-operation-dialog/kyfiledialogrename.cpp" line="131"/>
         <location filename="../file-operation-dialog/kyfiledialogrename.cpp" line="173"/>
         <location filename="../file-operation-dialog/kyfiledialogrename.cpp" line="240"/>
         <source>OK</source>
-        <translation type="unfinished"></translation>
+        <translation>ຢືນຢັນ</translation>
     </message>
     <message>
         <location filename="../file-operation-dialog/kyfiledialogrename.cpp" line="69"/>
         <source>File &quot;%1&quot;</source>
-        <translation type="unfinished"></translation>
+        <translation>ໄຟລ໌ &quot;%1&quot;</translation>
     </message>
     <message>
         <source>Filename too long. </source>
@@ -172,17 +172,17 @@
     <message>
         <location filename="../file-operation-dialog/kyfiledialogrename.cpp" line="71"/>
         <source>Please choose the following processing method:</source>
-        <translation type="unfinished"></translation>
+        <translation>ກະລຸນາເລືອກວິທີການປະມວນຜົນ:</translation>
     </message>
     <message>
         <location filename="../file-operation-dialog/kyfiledialogrename.cpp" line="292"/>
         <source>truncate interval</source>
-        <translation type="unfinished"></translation>
+        <translation>ໄລຍະຕັດ</translation>
     </message>
     <message>
         <location filename="../file-operation-dialog/kyfiledialogrename.cpp" line="294"/>
         <source>Explanation: Truncate the portion of the file name that exceeds %1 %2 and select </source>
-        <translation type="unfinished"></translation>
+        <translation>ຄຳອະທິບາຍ: ຕັດສ່ວນຂອງຊື່ໄຟລ໌ທີ່ເກີນ %1 %2, ເລືອກ </translation>
     </message>
     <message>
         <source>Explanation: Truncate the portion of the file name that exceeds %1 %2 and select</source>
@@ -195,7 +195,7 @@
     <message>
         <location filename="../file-operation-dialog/kyfiledialogrename.cpp" line="298"/>
         <source>&quot;.</source>
-        <translation type="unfinished"></translation>
+        <translation>”.</translation>
     </message>
     <message>
         <source>Description: Save the file to &quot;%1/扩展&quot;.</source>
@@ -211,88 +211,88 @@
     <message>
         <location filename="../kyfiledialog.cpp" line="282"/>
         <source>Go Back</source>
-        <translation type="unfinished"></translation>
+        <translation>ກັບຄືນ</translation>
     </message>
     <message>
         <location filename="../kyfiledialog.cpp" line="289"/>
         <source>Go Forward</source>
-        <translation type="unfinished"></translation>
+        <translation>ໄປຂ້າງຫນ້າ</translation>
     </message>
     <message>
         <location filename="../kyfiledialog.cpp" line="295"/>
         <source>Cd Up</source>
-        <translation type="unfinished"></translation>
+        <translation>ຂຶ້ນ</translation>
     </message>
     <message>
         <location filename="../kyfiledialog.cpp" line="302"/>
         <source>Search</source>
-        <translation type="unfinished"></translation>
+        <translation>ຄົ້ນຫາ</translation>
     </message>
     <message>
         <location filename="../kyfiledialog.cpp" line="310"/>
         <source>View Type</source>
-        <translation type="unfinished"></translation>
+        <translation>ປະເພດມຸມມອງ</translation>
     </message>
     <message>
         <location filename="../kyfiledialog.cpp" line="321"/>
         <source>Sort Type</source>
-        <translation type="unfinished"></translation>
+        <translation>ประเภทธຳນັດ</translation>
     </message>
     <message>
         <location filename="../kyfiledialog.cpp" line="326"/>
         <location filename="../kyfiledialog.cpp" line="444"/>
         <source>Maximize</source>
-        <translation type="unfinished"></translation>
+        <translation>ຂະຫຍາຍໃຫ້ເຕັມ</translation>
     </message>
     <message>
         <location filename="../kyfiledialog.cpp" line="340"/>
         <source>Close</source>
-        <translation type="unfinished"></translation>
+        <translation>ປິດ</translation>
     </message>
     <message>
         <location filename="../kyfiledialog.cpp" line="441"/>
         <source>Restore</source>
-        <translation type="unfinished"></translation>
+        <translation>ກູ້ຄືນ</translation>
     </message>
     <message>
         <location filename="../kyfiledialog.cpp" line="1058"/>
         <source>Name</source>
-        <translation type="unfinished"></translation>
+        <translation>ຊື່ໄຟລ໌</translation>
     </message>
     <message>
         <location filename="../kyfiledialog.cpp" line="1061"/>
         <location filename="../kyfiledialog.cpp" line="1698"/>
         <source>Open</source>
-        <translation type="unfinished"></translation>
+        <translation>ເປີດ</translation>
     </message>
     <message>
         <location filename="../kyfiledialog.cpp" line="1062"/>
         <location filename="../kyfiledialog.cpp" line="1071"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>ຍົກເລີກ</translation>
     </message>
     <message>
         <location filename="../kyfiledialog.cpp" line="1066"/>
         <source>Save as</source>
-        <translation type="unfinished"></translation>
+        <translation>ບັນທຶກເປັນ</translation>
     </message>
     <message>
         <location filename="../kyfiledialog.cpp" line="1068"/>
         <source>New Folder</source>
-        <translation type="unfinished"></translation>
+        <translation>ສ້າງໄຟລ໌ໃໝ່</translation>
     </message>
     <message>
         <location filename="../kyfiledialog.cpp" line="1070"/>
         <location filename="../kyfiledialog.cpp" line="1702"/>
         <source>Save</source>
-        <translation type="unfinished"></translation>
+        <translation>ບັນທຶກ</translation>
     </message>
     <message>
         <location filename="../kyfiledialog.cpp" line="1106"/>
         <location filename="../kyfiledialog.cpp" line="1107"/>
         <location filename="../kyfiledialog.cpp" line="1109"/>
         <source>Directories</source>
-        <translation type="unfinished"></translation>
+        <translation>ໄຟລ໌</translation>
     </message>
     <message>
         <location filename="../kyfiledialog.cpp" line="1299"/>
@@ -301,37 +301,37 @@
         <location filename="../kyfiledialog.cpp" line="2907"/>
         <location filename="../kyfiledialog.cpp" line="2914"/>
         <source>Warning</source>
-        <translation type="unfinished"></translation>
+        <translation>ຄຳເຕືອນ</translation>
     </message>
     <message>
         <location filename="../kyfiledialog.cpp" line="1299"/>
         <source>exist, are you sure replace?</source>
-        <translation type="unfinished"></translation>
+        <translation>ມີຢູ່ແລ້ວ, ທ່ານຕ້ອງການແທນທີ່ບໍ່?</translation>
     </message>
     <message>
         <location filename="../kyfiledialog.cpp" line="2064"/>
         <source>NewFolder</source>
-        <translation type="unfinished"></translation>
+        <translation>ສ້າງໄຟລ໌ໃໝ່</translation>
     </message>
     <message>
         <location filename="../kyfiledialog.cpp" line="2405"/>
         <source>Undo</source>
-        <translation type="unfinished"></translation>
+        <translation>ຍົກເລີກ</translation>
     </message>
     <message>
         <location filename="../kyfiledialog.cpp" line="2412"/>
         <source>Redo</source>
-        <translation type="unfinished"></translation>
+        <translation>ເຮັດຄືນໃໝ່</translation>
     </message>
     <message>
         <location filename="../kyfiledialog.cpp" line="2609"/>
         <source>warn</source>
-        <translation type="unfinished"></translation>
+        <translation>ຄຳເຕືອນ</translation>
     </message>
     <message>
         <location filename="../kyfiledialog.cpp" line="2609"/>
         <source>This operation is not supported.</source>
-        <translation type="unfinished"></translation>
+        <translation>ບໍ່ຮອງຮັບການດຳເນີນການນີ້.</translation>
     </message>
     <message>
         <location filename="../kyfiledialog.cpp" line="2878"/>
@@ -339,7 +339,7 @@
         <location filename="../kyfiledialog.cpp" line="2907"/>
         <location filename="../kyfiledialog.cpp" line="2914"/>
         <source>File save failed! </source>
-        <translation type="unfinished"></translation>
+        <translation>ບັນທຶກໄຟລ໌ບໍ່ສຳເລັດ! </translation>
     </message>
 </context>
 <context>
@@ -358,52 +358,52 @@
     <message>
         <location filename="../uikyfiledialog.cpp" line="237"/>
         <source>File Name</source>
-        <translation type="unfinished"></translation>
+        <translation>ຊື່ໄຟລ໌</translation>
     </message>
     <message>
         <location filename="../uikyfiledialog.cpp" line="241"/>
         <source>Modified Date</source>
-        <translation type="unfinished"></translation>
+        <translation>ວັນທີແກ້ໄຂ</translation>
     </message>
     <message>
         <location filename="../uikyfiledialog.cpp" line="245"/>
         <source>File Type</source>
-        <translation type="unfinished"></translation>
+        <translation>ປະເພດຂອງໄຟລ໌</translation>
     </message>
     <message>
         <location filename="../uikyfiledialog.cpp" line="249"/>
         <source>File Size</source>
-        <translation type="unfinished"></translation>
+        <translation>ຂະໜາດຂອງໄຟລ໌</translation>
     </message>
     <message>
         <location filename="../uikyfiledialog.cpp" line="253"/>
         <source>Original Path</source>
-        <translation type="unfinished"></translation>
+        <translation>ເສັ້ນທາງຕົ້ນສະບັບ</translation>
     </message>
     <message>
         <location filename="../uikyfiledialog.cpp" line="262"/>
         <source>Descending</source>
-        <translation type="unfinished"></translation>
+        <translation>ລົງ</translation>
     </message>
     <message>
         <location filename="../uikyfiledialog.cpp" line="267"/>
         <source>Ascending</source>
-        <translation type="unfinished"></translation>
+        <translation>ຈັດຈຳນວນຂຶ້ນ</translation>
     </message>
     <message>
         <location filename="../uikyfiledialog.cpp" line="273"/>
         <source>Use current sorting for all folders</source>
-        <translation type="unfinished"></translation>
+        <translation>ໃຊ້ການຈັດລຽງທີ່ກຳນົດໃນໄຟລ໌ທັງໝົດ</translation>
     </message>
     <message>
         <location filename="../uikyfiledialog.cpp" line="293"/>
         <source>List View</source>
-        <translation type="unfinished"></translation>
+        <translation>ມຸມມອງລາຍຊື່</translation>
     </message>
     <message>
         <location filename="../uikyfiledialog.cpp" line="294"/>
         <source>Icon View</source>
-        <translation type="unfinished"></translation>
+        <translation>ມຸມມອງຮູບພາບ</translation>
     </message>
 </context>
 </TS>
